# Generated by Django 4.1.2 on 2022-10-19 12:00

from django.db import migrations, models
import services.validators


class Migration(migrations.Migration):

    dependencies = [
        ('accounts', '0002_alter_user_unique_together'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='phone_number',
            field=models.CharField(error_messages={'unique': 'Пользователь с таким номером телефона уже существует.'}, help_text="Обязательно. Максимально допустимое кол-во символов - 18. Цифры и символы '()', '+' и '_'.", max_length=12, unique=True, validators=[services.validators.PhoneNumberValidator()], verbose_name='Номер телефона'),
        ),
    ]
